plugins/repository-azure/src/main/java/org/elasticsearch/repositories/azure/AzureBlobStore.java

```diff
@@ -99,42 +95,42 @@ public void delete(BlobPath path) {
     public void close() {
     }
 
-    public boolean doesContainerExist(String container)
+    public boolean doesContainerExist()
     {
         return this.client.doesContainerExist(this.clientName, this.locMode, container);
     }
 
-    public void deleteFiles(String container, String path) throws URISyntaxException, StorageException
+    public void deleteFiles(String path) throws URISyntaxException, StorageException
```

Comment:
I don't think this method is used anymore. So, maybe you should clean it up as well while you are at it.